Source/BSN.Commons/Infrastructure/MessageBroker/EventContracts/EventAggregator/EventModels/IEventDataModel.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace BSN.Commons.Infrastructure.MessageBroker.EventContracts.EventAggregator.EventModels
+{
+    public interface IEventDataModel
```

Comment:
All intefaces must be clear and comprehensive documentation that describes responsibilities and concepts

Reply:
Please make better comments for concept of interfaces try to representing